### ISSUE TITLE
Unrecognized escape sequence error in pattern-based record struct definition

### ISSUE THREAD

**Issue by koddek** [dtanglr/Primitively#12]
When I try to build this code:
```cs
public const string Pattern2 = "^[a-zA-Z0-9._%+-]{1,64}@[a-zA-Z0-9.-]{1,255}\\.[a-zA-Z]{2,}$";
public const string Pattern3 = """^[a-zA-Z0-9._%+-]{1,64}@[a-zA-Z0-9.-]{1,255}\.[a-zA-Z]{2,}$""";

[String(minLength: 6, maxLength: 99,Pattern = Pattern1)]
public readonly partial record struct Email;
```
I get this error for both ways of arranging the pattern:
> 0>PrimitiveRepository.g.cs(35,200): Error CS1009 : Unrecognized escape sequence

**Comment by dtanglr** [comment]
Thanks @koddek. I will take a look.

**Comment by dtanglr** [comment]
@koddek I've just published version 1.4.22 which contains a fix for your issue. I've included a unit test that covers your problematic regex (see the release/1.4.22 branch).

`[String(minLength: 6, maxLength: 99, Pattern = @"^[a-zA-Z0-9._%+-]{1,64}@[a-zA-Z0-9.-]{1,255}\.[a-zA-Z]{2,}$", Example = "test@example.com", ImplementIValidatableObject = true)]
public readonly partial record struct Email;`
